src/lips.js

```diff
@@ -7950,7 +7950,7 @@ var global_env = new Environment({
                     if (bindings) {
                         /* istanbul ignore next */
                         if (is_debug()) {
-                            console.log(JSON.stringify(symbolize(bindings), true, 2));
+                            console.log(JSON.stringify(symbolize(bindings), true, 2)); // @jcubic can you enter what this console log returns here
```

Comment:
This is yet another debug code for syntax-rules (hygienic macros) that don't work 100% correct.

Reply:
Should be deleted now. I made a new commit.